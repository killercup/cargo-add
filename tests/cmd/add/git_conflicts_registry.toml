bin.name = "cargo-add"
args = ["add", "my-package", "--git", "https://github.com/dcjanus/invalid", "--registry", "alternative"]
status.code = 2
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
