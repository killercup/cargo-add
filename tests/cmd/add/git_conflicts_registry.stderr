error: The argument '--git <URI>' cannot be used with '--registry <REGISTRY>'

USAGE:
    cargo add --git <URI> --upgrade <METHOD> <CRATE>...

For more information try --help
